docs/articles/documentation/test-api/actions/click.md

```diff
@@ -0,0 +1,48 @@
+---
+layout: docs
+title: Click
+permalink: /documentation/test-api/actions/click.html
+---
+# Click
+
+Clicks a webpage element.
```

Comment:
We write "clicks element", "double clicks" etc, but we just simulate this actions. I know we have note in the end of the action description, but perhaps it would be better honestly write "simulates element click"?